libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -102,6 +105,9 @@ class AC_WPNav
     /// get_loiter_target - returns loiter target position
     const Vector3f& get_loiter_target() const { return _pos_control.get_pos_target(); }
 
+    /// get_loiter_angle_max - returns the maximum pilot commanded angle in degrees
```

Comment:
degrees -> centi-degrees

Reply:
txs.  fixed get-loiter-angle-max comments in .h file as well.